test/manual/implement_test_maps.R

```diff
@@ -0,0 +1,27 @@
+#' In this code we create maps for all 251 iso3 codes
+#' For each map a random sample of points is drawn over the bounding box
+#' of the admin 0 boundary and plotted.
+
+# Implement map creation
+box::use(../../src/images/maps/map_test)
+box::use(purrr[map])
```

Comment:
yea that seems like a good practice.... global - space - local.

So your saying your prefer seeing... `purrr$map()` than `map()`? Find we/ me to use `purrr$map` especially in the context of signals where is the `$ `is used already used alot w/ standard libraries (i.e `purrr$`, `dplyr$`, `sf$`).

but maybe for other repos/work we might want to think of the best approach because a the end of the day `package$func` is the same as`package::func` in-terms of straight readability. With the box version making our overall work much harder to reproduce and understand for the majority of R-users. I'd say there probably is a sweet spot for typical data analysis projects where you can use a combo in a smart way...but thats for another discussion :-)